tests/scripts/ci.py

```diff
@@ -24,12 +24,15 @@
 import getpass
 import inspect
 import argparse
+import json
+import shutil
 import grp
 import subprocess
 from pathlib import Path
 from typing import List, Dict, Any, Optional
 
 REPO_ROOT = Path(__file__).resolve().parent.parent.parent
+SCRIPT_DIR = REPO_ROOT / ".cipy-scripts"
```

Comment:
what happens if someone runs this command concurrently? should we name like `.ci-py-scripts/<pid>`?

Reply:
Since this is for people to run interactively and these files are consumed almost right after they are created I don't think it's a big deal to say that concurrent runs are just unsupported to keep things simple (at least for now)

Reply:
sg